include/aws/iotdevice/secure_tunneling.h

```diff
@@ -67,7 +67,7 @@ enum aws_secure_tunnel_message_type {
 
     /**
      * ConnectionReset messages convey that the connection has ended, either in error, or closed intentionally for the
-     * tunnel peer.
+     * tunnel peer. These should not be manually sent from either Destination or Source clients.
```

Comment:
Nitpick: If this is for testing only, I'd add a note about it here too. Something like "This is only used for testing internally" or something, so the customer knows that they never should see this in a real world application.